tensorrt_llm/_torch/speculative/eagle3.py

```
@@ -643,97 +671,113 @@ def _forward_linear_draft_loop(self, inputs, attn_metadata, spec_metadata,
+                            start_ids_gen = (
+                                spec_metadata.batch_indices_cuda[:num_gens] *
+                                (runtime_draft_len + 1)).long()
+                            gather_ids_gen = (
+                                start_ids_gen +
+                                num_accepted_tokens[num_contexts:] - 1 +
+                                attn_metadata.num_ctx_tokens)
+                            gather_ids = torch.concat([
+                                spec_metadata.gather_ids[:num_contexts],
+                                gather_ids_gen
+                            ],
+                                                      dim=0)
```

Flake8 (7.3.0)

[error] continuation line over-indented for hanging indent

(E126)